sections/main-product.liquid

```diff
@@ -724,6 +725,12 @@
       "default": true,
       "label": "t:sections.main-product.settings.enable_sticky_info.label"
     },
+    {
+      "type": "checkbox",
+      "id": "hide_variants",
+      "default": false,
+      "label": "Hide inactive variants"
```

Comment:
```suggestion
      "default": true,
      "label": "Show variant media"
```

@gretchen-willenberg we need to add a setting in the Product template because we want to give the ability to only show the images in the gallery that correspond to the currently selected variant.

In Narrative, we have: `Hide variant media in gallery`

I am wondering if we should give the positive form of the setting? By default, merchants would want to show all their images so this setting would be true by default if this is the direction we take.

Reply:
In deciding if it should be positive (Show) or negative (Hide), I think most of our checkboxes are opt-in/unchecked. (?) So...if merchants by default want to show all their images, we want to give them the choice to show only the selected variant's.  I think that can still use the positive Show and narrow down what is being shown. [All in the Google doc, too](https://docs.google.com/document/d/14Aa-twJrAWSo95v7RwTF0M2Pk5xJE3t10GJfmcesWBU/edit?usp=sharing)

`Show media for only the selected variant`
`Show only the selected variant’s image(s)`
`When a variant is selected, show only its media`  _I like how this explains the action._

Tried a few similar versions with `Hide` just to see:
`Hide media of unselected variants`
`Hide other variants’ media when a variant is selected`

Reply:
The use of `only` worries me a bit. As when checked, the setting will show not only the selected variant's image but the other medias that aren't linked to variants as well. 
I find `Hide media of unselected variants` to be pretty clear. Just my two cents 😅